src/main/java/no/unit/nva/cristin/projects/CristinApiClient.java

```diff
@@ -75,21 +67,24 @@ protected URL generateQueryProjectsUrl(Map<String, String> parameters) throws Ma
 
     protected URL generateGetProjectUrl(String id, String language) throws MalformedURLException, URISyntaxException {
         URI uri = new URIBuilder()
-                .setScheme(HTTPS)
-                .setHost(CRISTIN_API_HOST)
-                .setPath(CRISTIN_API_PROJECTS_PATH + id)
-                .addParameter("lang", language)
-                .build();
+            .setScheme(HTTPS)
```

Comment:
not in scope and not very important but is there a possibility to build the URIs using the java constructors new URI()? The less libraries we use the faster the startup time. Of course if we need to use something, we should use it. Anyhow, I am just seeing it because of the linting, and it is outside the scope of this PR.

Reply:
I realise some of my comments are not in scope — just looking at this code made me react in general, this isn't your fault @torarnet :D

Reply:
Seems there is no easy way to do it and would also make code less readable as built in URI uses parameters like scheme and host in constructor only and there were no easy way to add query params, (it only takes in a string, we need to concatenate ourselves). We can do it but then we have to do a little more work with the query params, maybe in another pull request?

Reply:
Do not worry about it now